float64 sonar_1
float64 sonar_2
float64 sonar_3
float64 sonar_4
float64 sonar_5
float64 sonar_6
float64 sonar_7
float64 sonar_8
float64 sonar_9
